src/flick/like/views.py

```diff
@@ -5,12 +5,22 @@
 from api.utils import success_response
 from comment.models import Comment
 from comment.serializers import CommentSerializer
+from notification.models import Notification
 from rest_framework import generics
 
 
 class LikeView(generics.GenericAPIView):
     permission_classes = api_settings.CONSUMER_PERMISSIONS
 
+    def _create_comment_like_notification(self, from_user, to_user, comment):
+        from_profile = Profile.objects.get(user=from_user)
+        notif = Notification()
+        notif.notif_type = "comment_like"
+        notif.from_user = from_profile
+        notif.to_user = to_user
```

Comment:
could we make sure sure if i like my own comment, i don't get a notification lol so like to_user != from_user

Reply:
good catch

Reply:
oki! that makes sense!